clarity/tests/collateral-rebalancing-pool_test.ts

```diff
@@ -585,7 +585,7 @@ Clarinet.test({
         
         // arbtrageur attepmts to retreive back with zero value
         result = CRPTest.reducePositionYield(deployer, wbtcAddress, usdaAddress, expiry, yieldwbtcAddress, 0);        
-        result.expectErr().expectUint(3000) 
+        result.expectErr().expectUint(3) 
```

Comment:
Aside: consider defining constants for all expected error codes, matching the error constant names found in the contracts.

```js
const ErrorCodes = {
 SomeErrorStatus: 3
};
```

Safer to reduce magic numbers whenever possible.

Reply:
Agreed!